modules/eco-self-node-remediation-operator-about.adoc

```diff
@@ -89,12 +89,12 @@ The Self Node Remediation Operator also creates the `SelfNodeRemediationTemplate
 
 `ResourceDeletion`:: This remediation strategy removes the pods and associated volume attachments on the node rather than the node object. This strategy helps to recover workloads faster. `ResourceDeletion` is the default remediation strategy.
 
-`NodeDeletion`:: This remediation strategy removes the node object.
+`NodeDeletion`:: This remediation strategy is deprecated and will be removed in a future release. It should not be selected, but if it is it operates in the same way as `ResourceDeletion` does on the node object.
```

Comment:
@prabinovRedhat  Your statement is correct, but IIUC this is also what the text say.
Can you explain in more details what do you find incorrect in the PR's text ?

Reply:
@mshitrit  yes, sure I can explain. When I see this phrase: "It should not be selected, but if it is"- it sounds like we shouldn't use it but if we use.. and it sound's incorrect for me because user can't really use it, even if he will (actually will be used the Resource Deletion strategy instead).. so it's kind of confusing for me, but maybe I'm wrong..

Reply:
@prabinovRedhat I understand what you mean re confusion. Maybe it's best to remove the 2nd sentence 'It should not be selected, but if it is it operates in the same way as `ResourceDeletion` does on the node object.'

Reply:
I think that we should let the user know that we are using `ResourceDeletion` in any case (I wouldn't want the user to be surprised by that) but I agree we can improve the phrasing :smile: 

@prabinovRedhat , @ogradyp How about something like this:
`NodeDeletion`:: This remediation strategy is deprecated and will be removed in a future release. In the current release `ResourceDeletion` will be used even if `NodeDeletion` strategy is selected.

Reply:
Thanks @mshitrit I think this new phrasing works well.

Reply:
Thanks Michael . I have applied this change to read.
NodeDeletion
This remediation strategy is deprecated and will be removed in a future release. In the current release, the `ResourceDeletion` strategy will be used even if the `NodeDeletion` strategy is selected.